sites/offshore-mag/server/templates/website-section/resources/maps-posters.marko

```diff
@@ -47,7 +47,7 @@ $ const { id, alias, name, pageNode } = data;
         }
         fragment-name="content-list"
         query-name="website-scheduled-content"
-        query-params={ sectionId: id, limit: 14, skip: 12 }
+        query-params={ sectionId: id, limit: 14}
```

Comment:
This PR is correct, however, the load more should not fire this quickly. A `load-more-flow` should be added before the full load more block. Otherwise, because the page is only a header, the load more will fire immediately on page load, causing two page view events to fire. Take a look at this PR for example of fixing this: https://github.com/base-cms-websites/industrial-media/pull/16

Hit me up on Slack if you have questions.

Reply:
Is this what you meant?

Reply:
Yep! Looks good.